Twig/Extension.php

```diff
@@ -27,17 +27,23 @@ class Extension extends AbstractExtension
      */
     private $trackerPath;
 
+    /**
+     * @var bool
+     */
+    private $disableCookies;
+
     /**
      * @var array
      */
     private $paqs = [];
 
-    public function __construct(bool $disabled, string $siteId, string $piwikHost, string $trackerPath)
+    public function __construct(bool $disabled, string $siteId, string $piwikHost, string $trackerPath, ?bool $disableCookies)
```

Comment:
I'd set `false` as the default value for the parameter here, so this part stays BC. You also don't need to change the tests above.